tests/data_files/cert_md2.csr

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
I've build the library with the default configuration + MD2 + MD4 and then I've run "make -C tests/data_files" and I end up with other files that are neither ignored not checked in:

cli2.req.sha256
server1.cert_type.der
server1.key_usage.der
server1.v1.der
server2.req.sha256
server7.csr
test-ca.req.sha256
test-ca.req_ec.sha256
test-ca2.req.sha256
test-int-ca.csr

Should we commit or ignore those ones?

Reply:
Yes. I don't know which, and I consider them out of scope of this PR. I only meant to fix the cases that I encountered.

Reply:
Filed as https://github.com/ARMmbed/mbedtls/issues/3715

Reply:
Thanks for filing an issue for that.